src/browsercheck.js

```diff
@@ -20,14 +19,14 @@ function getBrowserName(agent) {
 }
 
 function getBrowserVersionFromUserAgent(agent) {
-  var browserName = getBrowserName(agent);
+  var browserName = getBrowserName(agent).toLowerCase();
   var version = (browserName === 'ios_saf'
     ? agent.os.version
     : agent.browser.version || agent.os.version || ''
   ).split('.');
   while (version.length > 0) {
     try {
-      return browserslist(browserName + ' ' + version.join('.'))[0];
+      return browserName + ' ' + version.join('.');
```

Comment:
```suggestion
      return `${browserName}  ${version.join('.')}`;
```

Reply:
Not for this file, which needs to stay es5-compatible.